providers/gov/nih/normalizer.py

```diff
@@ -71,7 +71,9 @@ def format_org_address(self, doc):
 
 
 class AgentRelation(Parser):
-    related = Delegate(AwardeeAgent, ctx)
+    schema = 'AgentWorkRelation'
```

Comment:
Can we say anything more specific about this relationship? I don't know if any existing types fit, but it might be worth adding "grant applicant" (if that's accurate?) or something.